client/package.json

```diff
@@ -18,6 +18,11 @@
   "devDependencies": {
     "@types/node": "^22.5.5",
     "@types/react": "^18.3.8",
+    "autoprefixer": "^10.4.20",
+    "postcss": "^8.4.47",
+    "prettier": "^3.3.3",
+    "prettier-plugin-tailwindcss": "^0.6.8",
```

Comment:
See if we can delete these. Prettier should be installed globally, ie. the root.